.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
```

Comment:
@Noshi96 tak na przyszłosc, to poprawki do pull requestu powinienes wykonać na poprzednim, nie ma potrzeby tworzenia nowego, pracujemy na jednym, my dajemy komentarze, Ty poprawiasz i potem mergujemy na koncu. Zostanmy juz na tym pull requescie i kolejne poprawki prosze rob juz tutaj, poprzedni moze byc zamkniety